### DISCUSSION TITLE
How do you get brain badge?

### DISCUSSION THREAD

**Discussion by yusuftemel** [community/community#27918] [Discussions] [answered]
How do you get useful badges in Github?

**Comment by ok-nick** [comment]
By having your discussion answer accepted at least 2 times.

Check out this repo for a list of all the badges and how to earn them:
https://github.com/Schweinepriester/github-profile-achievements

**Reply by yusuftemel** [reply]
thanks for your reply. [ok-nick](https://github.com/ok-nick)

**Comment by MrSarno** [comment, marked as answer]
Hey there, @yusuftemel!

There’s a guide on the achievements & how to earn them [here](https://github.com/Schweinepriester/github-profile-achievements/blob/main/README.md).

For the Galaxy Brain achievement, your responses in discussions need to be selected as the answers.

Requirements for the responses to count towards your achievement progress include;

- The answers can’t be in discussions you started yourself.
- The discussions have to belong to public repositories.

If this comment has helped, please consider marking it as the answer so your question no longer appears in [the list](https://github.com/orgs/community/discussions?discussions_q=is%3Aunanswered) of unanswered questions needing help. Thanks!

**Reply by yusuftemel** [reply]
thanks a lot for your great reply [MrSarno](https://github.com/MrSarno)

**Reply by MrSarno** [reply]
Anytime, friend. 👍

**Comment by d4rkn3m3s1s** [comment]


**Reply by yusuftemel** [reply]
with my pleasure  [d4rkn3m3s1s](https://github.com/d4rkn3m3s1s)

**Reply by yusuftemel** [reply]
Unfortunately, it's not possible now.

**Comment by yusuftemel** [comment]
thanks all
